### DISCUSSION TITLE
Semi large log file cause problems?

### DISCUSSION THREAD

**Discussion by Nionor** [netalertx/NetAlertX#488] [Q&A]
Could a ~100MB log file cause problem? 
I had this weird behavior where the Maintenance page would hang, it opened up "fine" but when I tried to press any of the tabs nothing would happen.
After I removed it everything went back to normal.
Should I clean the log once in a while or maybe it was just some junk in the data itself that caused the problem?

**Comment by jokob-sk** [comment]
I know the current logging approach generates big log files if a verbose log level, such as 'verbose' or 'debug' is set. I'm thinking about two options:

1. Auto delete and keep only N lines (configurable) of logs
2. Purge file completely on user-defined schedule set to once a week by default

Any suggestions or concerns?

**Reply by Nionor** [reply]
Both are valid options I would think, maybe with the added function on option 2 to have them not purged but renamed with date or so for those that want to save all logs but not clogging up the web-ui.

**Reply by jokob-sk** [reply]
Implemented a basic log cleanup in a new maintenance plugin: https://github.com/jokob-sk/Pi.Alert/issues/494
